OTKit/otkit-typography-desktop/token.yml

```diff
@@ -1,115 +1,143 @@
 props:
-  font-family:
-    type: "raw"
-    value: "'BrandonText', -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Helvetica, Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji', 'Segoe UI Symbol'"
+  # The font name of OpenTable's desktop web brand font.
+  # =============================================
   font-family-brand:
     type: "raw"
     value: "BrandonText"
-  font-size-xsmall:
-    type: "size"
-    value: "{!font-size-xsmall}"
-  font-size-small:
-    type: "size"
-    value: "{!font-size-small}"
-  font-size-medium:
-    type: "size"
-    value: "{!font-size-medium}"
-  font-size-large:
-    type: "size"
-    value: "{!font-size-large}"
-  font-size-xlarge:
-    type: "size"
-    value: "{!font-size-xlarge}"
-  line-height-xsmall:
-    type: "size"
-    value: "{!line-height-xsmall}"
-  line-height-small:
-    type: "size"
-    value: "{!line-height-small}"
-  line-height-medium:
-    type: "size"
-    value: "{!line-height-medium}"
-  line-height-large:
-    type: "size"
-    value: "{!line-height-large}"
-  line-height-xlarge:
-    type: "size"
-    value: "{!line-height-xlarge}"
+
+  # Backward-compatible, generic font weights of OpenTable's desktop web brand font.
+  # Should only be used when composing font faces for the brand font.
+  # =============================================
   font-weight-normal:
     type: "raw"
     value: "normal"
   font-weight-medium:
-    type: "number"
+    type: "raw"
     value: "500"
   font-weight-bold:
     type: "raw"
     value: "bold"
 
-  heading-xlarge-font-size:
+
+  # The entire font family definition for OpenTable's desktop web.
+  # Apply to the appropriate root element such as <html>, <body> or `reactroot`.
+  # =============================================
+  font-family:
+    type: "raw"
+    value: "'BrandonText', -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Helvetica, Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji', 'Segoe UI Symbol'"
+
+
+  # Design decisions for font group sxmall-medium.
+  #
+  # =============================================
+  xsmall-medium-font-size:
     type: "size"
-    value: "{!font-size-xlarge}"
-  heading-xlarge-font-weight:
+    value: "12px"
+  xsmall-medium-font-weight:
```

Comment:
We have decided to call the 500 weights "-medium" instead of "-semibold" as the screenshot indicated because of a few reasons:

- What designers really mean is they want `font-weight: 500`. In desktop web context, medium = 500 and semibold = 600, which may be confusing.
- Most font families matches medium to 500. The only difference is iOS, where @mannionaco wants to use semibold (600) to mean medium because San Francisco font is too skinny / thin at the 500 level comparing to all other fonts. Again, this is an anomaly not a standard.